src/PKSim.Presentation/DTO/Mappers/PKSimPathToPathElementsMapper.cs

```diff
@@ -139,16 +139,26 @@ private void adjustDisplayPathForQuantity(PathElements pathElements, IQuantity q
          else if (quantity.IsNamed(PLASMA_UNBOUND))
             adjustDisplayPathForContainerObserver(pathElements, quantity);
 
+         else if (quantity.Name.StartsWith(TOTAL_FRACTION_OF_DOSE))
+            adjustDisplayPathForTotalFractionOfDose(pathElements, quantity);
+
+         //Container observers directly in a container located under organism
          else if (!pathElements.Contains(PathElement.BottomCompartment))
             adjustDisplayPathForContainerObserver(pathElements, quantity);
 
          else if (quantity.IsAnImplementationOf<IObserver>())
             adjustDisplayNameForMoleculeObserver(pathElements, quantity.DowncastTo<IObserver>());
       }
 
+      private void adjustDisplayPathForTotalFractionOfDose(PathElements pathElements, IQuantity quantity)
+      {
+         pathElements[PathElement.Container] = pathElements[PathElement.TopContainer];
```

Comment:
Because of how we organize our observers, this will never be super great but I think like that it's not bad
IT allows for discoverability under Organism:
![image](https://user-images.githubusercontent.com/1041237/55837295-f32c6500-5aee-11e9-9331-a438801a039f.png)